llvm-external-projects/iree-dialects/include/iree-dialects/Transforms/TransformMatchers.h

```diff
@@ -762,6 +769,30 @@ void makeSoftmaxMatcher(
     transform_ext::StructuredOpMatcher *&maxReductionCapture,
     transform_ext::StructuredOpMatcher *&softmaxRootCapture);
 
+struct MatchedConvolutionCaptures {
+  mlir::linalg::detail::ConvolutionDimensions convolutionDims = {};
+  SmallVector<int64_t> convolutionOpSizes = {};
+  SmallVector<int64_t> trailingOpSizes = {};
+  int64_t convolutionOutputElementalTypeBitWidth = 0;
+  int64_t maybeTrailingOutputElementalTypeBitWidth = 0;
+  int64_t maybeFillElementalTypeBitWidth = 0;
+};
+
+/// Creates a group of matchers for:
+///
+///     trailing(convolution(leading(), fill()))
```

Comment:
I don't see the matcher for `leading` in the implementation.

Reply:
I forgot to update the comment after removing the leading matcher at some earlier point. There is no leading matcher (at least for now).